### PR TITLE
[fix] fixing circleCI for AdaScale

### PR DESCRIPTION
**Pull request by min-xu-ai**
- ran black, isort, flake8, mypy

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Fixes # (issue).

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by msbaines — COMMENTED** [review]
Looks like there is still an issue:

ERROR: /home/circleci/fairscale/fairscale/optim/__init__.py Imports are incorrectly sorted.

File: fairscale/optim/adascale.py [hunk 1]
```diff
@@ -26,12 +26,13 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+# type: ignore
```
Comment (msbaines) [inline fairscale/optim/adascale.py:29]: Can you open an issue for making adascale type-checked.

**Review by msbaines — APPROVED** [review]
LGTM

File: fairscale/optim/adascale.py (hunk 1 above)
Comment (min-xu-ai) [reply fairscale/optim/adascale.py:29]: Sure. #143 filed. Will get to it after reproducibility work.

**Pull request merged**